server/miljø.ts

```diff
@@ -56,31 +35,34 @@ const devMiljø: Environment = {
     'https://familie.ekstern.dev.nav.no/familie/alene-med-barn/soknad/barnetilsyn',
   søknadSkolepengerUrl:
     'https://familie.ekstern.dev.nav.no/familie/alene-med-barn/soknad/skolepenger',
-  infoSideOvergangsstønadUrl: 'https://www.ekstern.dev.nav.no/overgangsstonad-enslig',
-  infoSideBarnetilsynUrl: 'https://www.ekstern.dev.nav.no/barnetilsyn-enslig',
-  infoSideSkolepengerUrl: 'https://www.ekstern.dev.nav.no/skolepenger-enslig',
-  saksbehandlingstiderUrl: 'https://www.ekstern.dev.nav.no/saksbehandlingstider',
+  infoSideOvergangsstønadUrl: 'https://www.nav.no/overgangsstonad-enslig',
+  infoSideBarnetilsynUrl: 'https://www.nav.no/barnetilsyn-enslig',
+  infoSideSkolepengerUrl: 'https://www.nav.no/skolepenger-enslig',
+  saksbehandlingstiderUrl: 'https://www.nav.no/saksbehandlingstider',
   utbetalingsoversiktUrl: 'https://www.nav.no/utbetalingsoversikt',
   utvidetBarnetrygdUrl: 'https://www.nav.no/utvidet-barnetrygd',
-};
+  klageUrl: 'https://www.nav.no/klage',
+  ...overskriv,
+});
 
-const prodMiljø: Environment = {
-  port: 8080,
-  søknadApiProxyUrl: 'http://familie-ef-soknad-api/familie/alene-med-barn/soknad-api/api',
+const lokaltMiljø: Environment = lagMiljø({
+  port: 3000,
+  søknadApiProxyUrl: brukDevApi()
+    ? 'https://familie-ef-soknad-api.intern.dev.nav.no/familie/alene-med-barn/soknad-api/api'
+    : 'http://localhost:8091/api',
+  oAuthCallbackUri: 'https://localhost:8080/familie/alene-med-barn/minside/oauth2/callback',
+});
+
+const devMiljø: Environment = lagMiljø({});
```

Comment:
For preprod er det kanskje greit å lenke til denne siden;
https://klage.intern.dev.nav.no/